centrifuge/src/main/java/io/github/centrifugal/centrifuge/Client.java

```diff
@@ -746,31 +748,22 @@ private void sendRefresh() {
                 if (Client.this.getState() != ClientState.CONNECTED) {
                     return;
                 }
-                if (error != null) {
```

Comment:
we have check in case error is null above, so this shouldn't happen here - that's why I've removed this check and else block

Reply:
oh right, the warning there was related to `ResultCallback` which was non-null at the time I was updating this code, thanks for the check, reverted